visual-js/visual-snapshots/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@saucelabs/visual-snapshots",
+  "description": "CLI which generates Visual snapshots from a data source such as pdf",
+  "version": "0.1.0",
+  "main": "lib/index.cjs",
+  "module": "lib/index.js",
```

Comment:
If you're using `tsc`, most likely you do not compile to `cjs` and `js`, but only to `js`. Can we change `main` to `"lib/index.js"` and remove `module`?